docs/network-drivers.md

```diff
@@ -75,8 +75,6 @@ HIL:
 
 * ``hil.ext.switches.dell``, which provides a driver for the Dell
   Powerconnect 5500 series switches.
-* ``hil.ext.switches.n3000``, which provides a driver for the Dell N3000
```

Comment:
Why are we removing this?

Reply:
idk how but it ended up there twice, so I am just removing the copy.

Reply:
Ah, makes sense (the other one didn't show in the diff, so I was confused...)